docs/en/user-guide/layerexplanations/layerexplanation.rst

```diff
@@ -71,10 +71,10 @@ If you want to export data to the VSA-DSS 2015 model, you have to manually add t
 
  .. figure:: images/interlis_export_class_organisation_subclass_checkjpg.jpg
 
- Alternative: You can use vw_organisation instead of od_organisation. vw_organisation has the subclasses integrated and must be loaded manually to the project.
+ Alternative: You can use tww_app.vw_organisation instead of organisation. tww_app.vw_organisation has the subclasses integrated and must be loaded manually to the project.
```

Comment:
There are no subclasses in organisation anymore with DSS2020. The distinction is now made with the attribute "organisation_type"
```

        Organisationstyp: MANDATORY (
          !!@ comment = "Abwasserverband"
          Abwasserverband,
          !!@ comment = "Teil einer Bundesverwaltung (z.B. Bundesamt für Umwelt, ASTRA, Armasuisse)"
          Bund,
          !!@ comment = "Gemeinde gemäss Bundesamt für Statistik"
          Gemeinde,
          !!@ comment = "Teil einer Gemeindeverwaltung, die eine Rolle übernimmt ((z.B. Geoinformation Stadt Bern) als Traegerschaft, Datenlieferant, etc. (aber nicht als Datenherr / (oeffentlicher) Eigentuemer)"
          Gemeindeabteilung,
          !!@ comment = "Genossenschaft oder Korporation: Körperschaft öffentlichen Rechts. Falls privaten Rechtes dann als Privat abbilden."
          Genossenschaft_Korporation,
          !!@ comment = "Teil einer kantonalen Verwaltung (z.B. Amt für Umweltschutz, Amt für Abwasserentsorgung)"
          Kanton,
          !!@ comment = "Privatperson oder Privatorganisation, welche im Rahmen der Entwässerungsplanung auftritt"
          Privat
        );
```